Workbooks/Virtual Machines/Availability/Server Availability.workbook

```diff
@@ -265,5 +274,9 @@
       "name": "query - 0"
     }
   ],
+  "fallbackResourceIds": [
+    "Azure Monitor"

```

Comment:
delete this section of `fallbackResourceIds` and `fromTemplateId`, the build is going to complain that they're there

Reply:
removed 

  "fallbackResourceIds": [
    "Azure Monitor"
  ],
  "fromTemplateId": "community-Workbooks/Virtual Machines/Availability",